pkg/v1/remote/referrers.go

```diff
@@ -81,6 +82,9 @@ func (f *fetcher) fetchReferrers(ctx context.Context, filter map[string]string,
 		} else if err != nil {
 			return nil, err
 		}
+	} else {
+		// TODO just returning "empty" here doesn't feel quite right -- maybe transport.CheckError again but this time letting http.StatusBadRequest return an error?
+		return empty.Index, nil
```

Comment:
Coming back to this almost a year later and feeling like a nice `fmt.Errorf` is probably more appropriate here.  Thoughts?  Maybe the variable needs to be opt-out instead of opt-in, also?  (`GGCR_REF_NO_FALLBACK` or `GGCR_REF_NEVER_FALLBACK` ?)